byron/ledger/executable-spec/test/Ledger/Relation/Properties.hs

```diff
@@ -0,0 +1,223 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE FlexibleContexts    #-}
+{-# LANGUAGE OverloadedStrings   #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Ledger.Relation.Properties
```

Comment:
Not for this PR, but in the near future we might want to move the `Relation` class and instances to a separate module. Just for us to think about it :)

Reply:
I've added a TODO for this